.ci/matrix-runtime-javas.yml

```diff
@@ -11,6 +11,7 @@ LS_RUNTIME_JAVA:
   - adoptopenjdk11
   - adoptopenjdk14
```

Comment:
Should we consider dropping `14` and `15` from the CI matrix? Elasticsearch only supports the most recent non-LTS version of the JVM (https://www.elastic.co/support/matrix#matrix_jvm), and only tests against that (https://github.com/elastic/elasticsearch/blob/master/.ci/matrix-runtime-javas.yml)

Reply:
I've updated to keep only Adopt, Zulu and OpenJDK 11 and 16 + java8